docs/operate/relayer/run-relayer.mdx

```diff
@@ -30,147 +36,189 @@ flowchart TB
-| `AWS_ACCESS_KEY_ID`     | The access key ID of your relayer's AWS IAM user.     |
-| `AWS_SECRET_ACCESS_KEY` | The secret access key of your relayer's AWS IAM user. |
   </TabItem>
 </Tabs>
 
-
 For chain-specific signers (i.e. to customize the key to use for each chain) take a look at the [configuration reference](../config-reference.mdx)
 
-### Installation
+## Start Relaying
+
+### Setup
 
 The recommended installation method for a production environment is using a Docker image.
 
 <Tabs groupId="docker">
   <TabItem value="docker" label="Docker image">
 
-To download the docker image, run:
+First download the docker image:
 
 ```bash
-docker pull gcr.io/abacus-labs-dev/hyperlane-agent:f44589e-20231130-114734
+docker pull gcr.io/abacus-labs-dev/hyperlane-agent:3bb4d87-20240129-164519
 ```
+
   </TabItem>
   <TabItem value="from-source" label="Building from source">
 
-First, clone the repo
+**Clone and setup**
+
+First, clone the Hyperlane monorepo:
 
 ```sh
 git clone git@github.com:hyperlane-xyz/hyperlane-monorepo.git
 ```
 
-And then follow the [setup instructions](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/README.md#setup) in the `rust` directory
+Then follow the [setup instructions](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/README.md) in the `rust` directory. This should setup `rustup` as well as Rosetta 2 if you are on Apple Silicon.
+
+```sh
+# install rustup
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# (apple silicon only) install rosetta 2
+softwareupdate --install-rosetta --agree-to-license
+```
+
+Build the Relayer:
+
+```sh
+cargo build --release bin relayer
+```
+
   </TabItem>
 </Tabs>
 
-### Start Relaying
+### Running the agent
+
+If the Relayer's keys have been configured with AWS KMS, you will have to provide the AWS access key and secret as environment variables.
+
+| Environment variable    | Description                                           |
+| ----------------------- | ----------------------------------------------------- |
+| `AWS_ACCESS_KEY_ID`     | The access key ID of your Relayer's AWS IAM user.     |
+| `AWS_SECRET_ACCESS_KEY` | The secret access key of your Relayer's AWS IAM user. |
 
-To run the relayer binary with configuration items specified via `CONFIG_FILES`, environment variables, or command line arguments, run:
+For a refresher, check out the [Agent Keys](../set-up-agent-keys.mdx) guide.
 
 <Tabs groupId="docker">
   <TabItem value="docker" label="Using Docker">
-Find the latest docker image and set it to the environment variable `$DOCKER_IMAGE`.
+
+Then start the container with the relevant arguments. For example, your configuration for AWS:
 
 ```sh
 docker run \
   -it \
-  --mount type=bind,source="$(pwd)"/hyperlane_db,target=/hyperlane_db \
-  $DOCKER_IMAGE \
-  ./relayer --db /hyperlane_db --relayChains ethereum,polygon,avalanche
+  -e AWS_ACCESS_KEY_ID=ABCDEFGHIJKLMNOP \
+  -e AWS_SECRET_ACCESS_KEY=xX-haha-nice-try-Xx \
```

Comment:
hahah

Reply:
just doing my bit to protect us from @laurenceday 🫡